amp.php

```diff
@@ -241,6 +241,28 @@ function _amp_incorrect_plugin_slug_admin_notice() {
 	add_action( 'admin_notices', '_amp_incorrect_plugin_slug_admin_notice' );
 }
 
+/**
+ * Print admin notice if the xdebug extension is loaded.
+ *
+ * @since T.B.D
+ */
+function _amp_xdebug_admin_notice() {
+	?>
+	<div class="notice notice-warning">
+		<p>
+			<?php
+			echo wp_kses_post(
```

Comment:
Since there are no tags appearing in the translation string, this could rather use `esc_html_e()`.